streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -230,7 +229,7 @@ private boolean waitOnState(final State targetState, final long waitMs) {
                         // it is ok: just move on to the next iteration
                     }
                 } else {
-                    log.debug("{} Cannot transit to {} within {}ms", logPrefix, targetState, waitMs);
+                    log.debug("Cannot transit to {} within {}ms", targetState, waitMs);
```

Comment:
@umesh9794 
nit: add space between the `value` and ms.

Reply:
@Kamal15 thanks for the review comments, the above format was there before my change so I'd prefer to keep as it is.